### DISCUSSION TITLE
How good is budibase as a landingpage/website for SEO ?

### DISCUSSION THREAD

**Discussion by MikeYates** [Budibase/budibase#11650] [General]
Im currently using wordpress for my business landing page, and plan to switch over to budibase for the landing page so that I have connectivity in data, and have everything controlled in one environment.

Main thing I worry about is the SEO of the landing page, my wordpress is configured so that my business has a high ranking SEO. What happens if I switch and use budibase instead? Should I do it?

**Comment by melohagan** [comment]
Hey @MikeYates 

Budibase is mostly designed with internal tools in mind, e.g. workflows, data entry, admin panels, etc. 

Not so much for front-end websites. I wouldn't consider it a suitable tool for a landing page.
